hydromt_sfincs/sfincs.py

```diff
@@ -2600,6 +2604,208 @@ def setup_wind_forcing(self, timeseries=None, magnitude=None, direction=None):
         )
         self.set_forcing(da, name="wnd")
 
+    def setup_wave_forcing(
+        self,
+        geodataset: Union[str, Path, xr.Dataset] = None,
+        timeseries: list = None, #Union[str, Path, pd.DataFrame] = None, #This should now be a list with dataframes, or could we do that anotherway?
```

Comment:
Agreed, this should be a list, but it can still be a list of strings, paths or in-memory dataframes, so this seems to better capture the options?:

```suggestion
        timeseries: List[Union[str, Path, pd.DataFrame]] = None, 
```